bytecode/tests/test_cfg.py

```diff
@@ -59,19 +60,32 @@ def test_constructor(self):
         self.assertBlocksEqual(code, [])
 
     def test_attr(self):
-        source = """
+        is_py2 = sys.version_info < (3,)
```

Comment:
You may use the six module to get six.PY2.

But at the same time, I would prefer to avoid depending on a third party module just for one constant...